charts/opentelemetry-demo/values.yaml

```diff
@@ -44,6 +45,11 @@ default:
           fieldPath: metadata.name
     - name: OTEL_RESOURCE_ATTRIBUTES
       value: service.name=$(OTEL_SERVICE_NAME),k8s.namespace.name=$(OTEL_K8S_NAMESPACE),k8s.node.name=$(OTEL_K8S_NODE_NAME),k8s.pod.name=$(OTEL_K8S_POD_NAME)
+  # list of environment variables which get applied to all components additional to the list provided with the "env" section.
+  # Use this section to override or add environment variables selectively 
+  envOverrides: []
+  #  - name: OTEL_K8S_NODE_NAME
+  #    value: "someConstantValue"
```

Comment:
The PR description says that this section is not supported or am I missing something?

Reply:
It is supported to override envs in the default section as part of the default section. So overwrites will apply for all components.
What is not supported is to use the envOverride in the component section to override an env which is introduced in the default section.
So the two env/envOverride pairs are isolated from each other as they are getting rendered independently

Reply:
@a-thaler looking at `_pod.tpl`, this field allows users to override `.Values.default.env` correct?  If that is the case, lets update the comment to be something like `# Allows overriding and additions to .Values.default.env`.  Hopefully that will make it clear that this field is only applied to `.Values.default.env` and nothing else.

The fact that `.Values.default.env` is applied to all components is already being documented with your new comment.

Reply:
Hi @TylerHelmuth , have adjusted the docu accordingly and bumped the versions once more